moma_mission/src/moma_mission/missions/piloting/manipulation_sim.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import rospy
+from nav_msgs.msg import Path
+from moma_mission.core import StateRosControl
+
+from moma_mission.utils.transforms import *
+from moma_mission.utils.robot import Robot
+
+class ValveManipulationState(StateRosControl):
```

Comment:
This class is very useful when for exampe the approximate location of the valve in known. So i wouldnt use it only in sim. For example, we can have a branching condition in the state machine and either go with this "ManipulationFromURDF" state of "ManipulationFromPerception" (you know what I mean)